test/fixtures/manifest/config/extra-files-glob.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Is this file used?

Reply:
All config files in that directory are turned into tests that verify that they adhere to the JSONSchema

Reply:
thx